Sources/AppcuesKit/AppcuesKit.docc/Tutorials/PermissionPrimers.tutorial

```diff
@@ -0,0 +1,120 @@
+            @Step {
+                Update your existing `Appcues.identify()` call to set a user property with the current camera permission status.
+                                
+                @Code(name: "MyAppAuth.swift", file: PermissionPrimers-01-03.swift, previousFile: PermissionPrimers-01-03-0.swift)
+            }
+            
+            @Step {
+                Configure the ``AppcuesKit/Appcues/analyticsDelegate`` of your Appcues instance.
+                
+                The analytics delegate is notified of every event handled within the Appcues iOS SDK, and allows us to detect the event that will trigger the system permission prompt.
+                                
+                @Code(name: "AppDelegate.swift", file: PermissionPrimers-01-04.swift, previousFile: PermissionPrimers-01-04-0.swift)
+            }
+            
+            @Step {
+                Filter for an event with value `request-permission-camera` and call `AVCaptureDevice.requestAccess`.
+                
+                A flow can trigger an event with this name to execute the system permission prompt.
+                                
+                @Code(name: "AppDelegate.swift", file: PermissionPrimers-01-05.swift, previousFile: PermissionPrimers-01-04.swift)
+            }
+            
+            @Step {
+                Update the completion callback to save the new state.
+                
+                                
+                @Code(name: "AppDelegate.swift", file: PermissionPrimers-01-06.swift, previousFile: PermissionPrimers-01-05.swift)
+            }
+
+        }
+    }
+        
+    @Section(title: "Create the flow in Appcues Mobile Builder") {
+        @ContentAndMedia {
+            Configure your app to report the permission status as an Appcues profile attribute and configure the Appcues iOS SDK to be able to trigger the permission prompt.
+        }
+        
+        @Steps {
+            @Step {
+                Create a new flow for your permission primer.
+                
+                @Image(source: PermissionPrimers-02-01.png, alt: "Mobile Builder Create Flow dialog")
+            }
+            
+            @Step {
+                Design your flow.
+                
+                Consider reviewing the [Human Interface Guidelines for Pre-alert screens](https://developer.apple.com/design/human-interface-guidelines/accessing-private-data#Pre-alert-screens).
+                
+                @Image(source: PermissionPrimers-02-02.png, alt: "Mobile permission primer flow")
+            }
+            
+            @Step {
+                Design your flow.
+                
+                Consider reviewing the [Human Interface Guidelines for Pre-alert screens](https://developer.apple.com/design/human-interface-guidelines/accessing-private-data#Pre-alert-screens).
+                
+                @Image(source: PermissionPrimers-02-02.png, alt: "Mobile permission primer flow")
+            }
```

Comment:
duplicate step 3 here with same content of step 2

Reply:
Oops, thanks. Do you think the 30 minute estimate is reasonable? It's not super helpful since it's probably different people doing different sections, but...

Reply:
I think 30 seems reasonable to read and find the right spots to adjust in the app and test it out. Fair point that the end designer of the flow would likely be someone else in the org later on down the line.